data/ignore_ids_safety_scan.json

```diff
@@ -1383,7 +1383,8 @@
                 "44716":"for shipping numpy 1.21.5 - the last available version for py3.7",
                 "44717":"for shipping numpy 1.21.5 - the last available version for py3.7",
                 "59398": "scipy bug - according to maintainer, not a vulnerability",
-                "59399": "scipy bug - no CVE attached"
+                "59399": "scipy bug - no CVE attached",
+                "64402":"Temporarily ignoring to patch CRITICAL vulnerabilties"
```

Comment:
Isn't `inference-neuron` for Inferentia 1 and `inference-neuronx` for inferentia 2 👀 ? @amzn-choeric

Reply:
Looks like it worked!

Reply:
Awesome!